.github/workflows/python-diff-lint.yml

```diff
@@ -0,0 +1,36 @@
+---
+name: Lint Python issues
+
+on:
+  push:
+  pull_request:
+    branches: [main]
+
+jobs:
+  python-lint-job:
+    runs-on: ubuntu-latest
+
+    permissions:
+      # required for all workflows
+      security-events: write
+
+    steps:
+      - name: Repository checkout
+        uses: actions/checkout@v3
+
+      - id: VCS_Diff_Lint
+        name: VCS Diff Lint
+        uses: fedora-copr/vcs-diff-lint-action@v1
+
+      - name: Upload artifact with detected defects in SARIF format
+        uses: actions/upload-artifact@v3
+        with:
+          name: VCS Diff Lint SARIF
+          path: ${{ steps.VCS_Diff_Lint.outputs.sarif }}
+        if: ${{ always() }}
```

Comment:
Is this needed? I think that even without the condition the jobs will run sequentially. [1]


[1] https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-not-requiring-successful-dependent-jobs

Reply:
Yes, per instructions in README.md: https://github.com/fedora-copr/vcs-diff-lint-action

The `vcs-diff-lint-action` can fail (some new Python errors created in PR => FAIL), and you still want to upload the SARIF.

Reply:
See this example run: https://github.com/devexp-db/distgen/actions/runs/4611524352/jobs/8151315618

Reply:
Right. Thanks for explanation.